radicle-cli/src/commands/issue.rs

```diff
@@ -171,13 +196,24 @@ pub fn run(options: Options, ctx: impl term::Context) -> anyhow::Result<()> {
     let repo = storage.repository(id)?;
     let mut issues = Issues::open(*signer.public_key(), &repo)?;
 
-    match options.op {
+    let op = match options.op {
+        Operation::Default => Operation::List {
+            assignee: Some(*profile.id()),
+        },
+        _ => options.op,
+    };
```

Comment:
Why do we need the `Default`? Can't `List` stay as the default operation?

Reply:
The behavior is defined as one of the 'dogfooding cli' issues https://github.com/radicle-dev/heartwood/issues/148

Reply:
Sorry, what's that got to do with this `Default` thing?

Reply:
Maybe I misunderstood the question, the task is to make 'rad issue' list only your assigned issues, which requires your profile ID.

Fetching the profile information happens in run(..), and it didnt feel right fetching it when processing the command's arguments.

Reply:
Ah, I think I see now. I was thinking that if the `assignee` was `None` you could default to the profile. But I see now that there are three distinct functionalities.

I'd suggest capturing the listing functionality in a function and not doing this pre-match to also avoid the panic below.

Reply:
How do you feel about this approach? https://github.com/radicle-dev/heartwood/pull/171/commits/422e28265f6a3fca1ef120390041d91a8b970891